### PR TITLE
Add support for using managed identity to manage AKS cluster resource

### PR DESCRIPTION
**Pull request by norshtein**
group

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by norshtein** [comment]
@yugangw-msft @zikalino could you please have a review?

**Comment by norshtein** [comment]
Friendly ping @yugangw-msft @zikalino

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -178,6 +178,9 @@
         - name: --enable-private-cluster
           type: string
           short-summary: (PREVIEW) Enable private cluster.
+        - name: --enable-managed-identity
+          type: bool
+          short-summary: (PREVIEW) Using managed identity to manage cluster resource group.
```
Comment (yugangw-msft) [inline src/aks-preview/azext_aks_preview/_help.py]: I suggest you call out the `system-assigned managed identity`. In future you can consider to expand the argument to take a list of used assigned ones

Comment (norshtein) [reply src/aks-preview/azext_aks_preview/_help.py]: Thanks for reminding, updated the help message.

**Comment by norshtein** [comment]
Also uploaded whl file to storage account and bumped version. Could you please help merge the PR @yugangw-msft ?

**Comment by norshtein** [comment]
friendly ping @yugangw-msft

**Pull request merged**